src/Core/Types/Types.csproj

```diff
@@ -25,6 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="GreenDonut" Version="2.0.0" />
+    <PackageReference Include="NJsonSchema" Version="9.13.37" PrivateAssets="True" />
```

Comment:
Will replace with a self-contained implementation of the [`XmlDocumentationExtensions`](https://github.com/RicoSuter/NJsonSchema/blob/master/src/NJsonSchema/Infrastructure/XmlDocumentationExtensions.cs).